libcore/gof-directory-async.vala

```diff
@@ -40,7 +40,7 @@ public class Async : Object {
     public int icon_size = 32;
 
     /* we're looking for particular path keywords like *\/icons* .icons ... */
-    public bool uri_contain_keypath_icons;
+    public bool uri_contain_keypath_icons = false;
```

Comment:
Can we change this variable name to `uri_contain`**`s`**`_keypath_icons` while we're here?

Reply:
Will do